### DISCUSSION TITLE
Add/edit Link should auto fill Display Text

### DISCUSSION THREAD

**Discussion by Christilut** [directus/directus#11589] [Feature Requests]
When selecting text in the wysiwig editor and pressing `Add/edit Link`, you get an empty `Display Text` field.
It would make perfect sense if that field is pre-filled with the selected text. Now you always have to type the same text by hand.

**Comment by azrikahar** [comment]
Good idea. Just opened a PR #11599 for this 👍

**Reply by Christilut** [reply]
Awesome!

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
